### DISCUSSION TITLE
Actividad 0 - Peres, Valentin

### DISCUSSION THREAD

**Discussion by PeresValentin** [frlp-utn-ingsoft/catedra#83] [Show and tell]
https://github.com/PeresValentin/tp0/tree/main/mysite

**Comment by ivanduva** [comment]
Buenas,
La actividad está bien. Los commits están bien y la app funciona correctamente.
Lo único es que este commit: https://github.com/PeresValentin/tp0/commit/8df40cef0737affc4ae516ad9ce140c82ae208ec lo hiciste desde la interfaz de github y en la consigna aclaramos que no se podía hacer. Al no ser generalizado no pasa nada, pero en las próximas actividades no lo vamos a dejar pasar.
La actividad queda aprobada, pero fuera de término. Vimos que los commits están hechos en horario de clase y no es la idea ya que nunca dispusimos un tiempo para realizar el trabajo durante la clase. Tenelo en cuenta para las próximas actividades.

Saludos

**Reply by RodrigoJacznik** [reply]
Ya había entregado. https://github.com/orgs/frlp-utn-ingsoft/discussions/81
